src/FlubuCore/Targeting/TargetTree.cs

```diff
@@ -131,7 +139,6 @@ public virtual void EnsureDependenciesExecuted(ITaskContextInternal taskContext,
         public virtual void ResetTargetTree()
         {
             _targets.Clear();
-            _executedTargets.Clear();
```

Comment:
Why was this line removed?

Reply:
Because .the next line is calling `ResetTargetExecutionInfo` method and that method has already do the same thing.

```
        public virtual void ResetTargetTree()
        {
            _targets.Clear();
            ResetTargetExecutionInfo();
            AddDefaultTargets();
        }

        public void ResetTargetExecutionInfo()
        {
            _executedTargets.Clear();
        }
```